lib/effects/move_effect.dart

```diff
@@ -22,13 +22,14 @@ class MoveEffect extends PositionComponentEffect {
   double _yDistance;
   double _yDirection;
 
-  MoveEffect({
-    @required this.destination,
-    @required this.speed,
-    this.curve,
-    isInfinite = false,
-    isAlternating = false,
-  }) : super(isInfinite, isAlternating);
+  MoveEffect(
+      {@required this.destination,
+      @required this.speed,
+      this.curve,
+      isInfinite = false,
+      isAlternating = false,
+      Function onComplete})
```

Comment:
Add a trailling comma here and keep the braces alongside the parenthesis (like it was previously).

Reply:
Fixed!